### PR TITLE
Add fingerprint for Sengled Water Leak Sensor

### PR DESCRIPTION
**Pull request by anzhuge**
1.add the init.lua file to the path ~\SmartThingsEdgeDrivers\drivers\SmartThings\zigbee-water-leak-sensor\src\sengled;
2.Modified ~\SmartThingsEdgeDrivers\drivers\SmartThings\zigbee-water-leak-sensor\fingerprints.yml file, a new device type E1L-G7K is added.

### CODE REVIEW

**Comment by ctowns** [comment]
@anzhuge the init.lua file in ~\SmartThingsEdgeDrivers\drivers\SmartThings\zigbee-water-leak-sensor\src\sengled is currently uploaded as a submodule. Please update this PR so that the init.lua file is present without a new submodule.

Also, please update the PR name to properly describe this PR, thanks! 

you can refer to this other sengled device PR for reference: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/374

File: drivers/SmartThings/zigbee-water-leak-sensor/fingerprints.yml [hunk 1]
```diff
@@ -94,3 +94,9 @@ zigbeeManufacturer:
     manufacturer: THIRDREALITY
     model: 3RWS18BZ
     deviceProfileName: water-battery
+  - id: "sengled E1L-G7K"
+    deviceLabel: Sengled Water Leak Sensor
+    manufacturer: sengled.
+    model: E1L-G7K
+    deviceProfileName: water-temp-battery-tempOffset
+
```
Comment (sjl2178) [inline drivers/SmartThings/zigbee-water-leak-sensor/fingerprints.yml:102]: nit. Please remove whitespace at the end of this file

File: drivers/SmartThings/zigbee-water-leak-sensor/fingerprints.yml [hunk 2]
```diff
@@ -94,3 +94,9 @@ zigbeeManufacturer:
     manufacturer: THIRDREALITY
     model: 3RWS18BZ
     deviceProfileName: water-battery
+  - id: "sengled E1L-G7K"
+    deviceLabel: Sengled Water Leak Sensor
+    manufacturer: sengled.
```
Comment (greens) [inline drivers/SmartThings/zigbee-water-leak-sensor/fingerprints.yml:99]: is this period on the end of the mfr string correct?

**Comment by greens** [comment]
please remove the unused directory: `drivers/SmartThings/zigbee-water-leak-sensor/src/sengled`

**Pull request closed without merging**